test/Blurb.spec.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-undef */
+describe('A blurb', function() {
+    const TEXT = 'Some willing text';
+
+    let page;
+
+    before(async function() {
+        page = await browser.newPage();
+        await page.goto('http://localhost:8080');
+        await page.type(SEL_TEXTAREA, TEXT);
+
+        let button = await page.$(SEL_SUBMIT);
+        await button.click();
+    });
+
+    after(async function() {
+        await page.close();
+    });
+
+    it('should appear', async function() {
+        await page.waitFor(SEL_CONTAINER);
+        expect(await page.$$(SEL_CONTAINER)).to.have.lengthOf(1);
+    });
```

Comment:
Make sure this style of test is consistent between this file and `Readout.spec.js`. I'm not sure that one approach is better than the other, but we should strive for a consistent style unless there's a good reason to break that rule of thumb.